editor/plugins/animation_blend_tree_editor_plugin.cpp

```diff
@@ -905,13 +948,13 @@ AnimationNodeBlendTreeEditor::AnimationNodeBlendTreeEditor() {
 	add_node->connect("about_to_popup", callable_mp(this, &AnimationNodeBlendTreeEditor::_update_options_menu));
 
 	add_options.push_back(AddOption("Animation", "AnimationNodeAnimation"));
-	add_options.push_back(AddOption("OneShot", "AnimationNodeOneShot"));
```

Comment:
Running clang tidy on the entire engine- this line was changed.

`2` is implicitly converted to `true` since the third constructor parameter of `AddOption` is a `bool`. Is this supposed to be `true`, or is `AddOption` supposed to take an `int`? Assuming the latter since the parameter has `count` in the name.

Reply:
Yeah, it was a small mistake.